### ISSUE TITLE
update Leiden+ documentation for supplied reason parallel

### ISSUE THREAD

**Issue by paregorios** [DCLP/dclpxsltbox#134]
per @HolgerEssler as originally posted to #120:

Probably the Text Leiden+ Documentation at http://papyri.info/editor/documentation?docotype=text should also be updated: The entry under "supplied parallel lost" (without specific content) could be merged with/replaced by the entry under "supplied parallel" (with example http://papyri.info/ddbdp/p.berl.leihg;2;39).

Breaking this task out as a separate ticket.

**Comment by paregorios** [comment]
blocked for #120

**Comment by paregorios** [comment]
no longer blocked by #120, which is resolved

**Comment by ryanfb** [comment]
Note that this should go in https://github.com/dclp/site-docs `leiden_plus.md`.
